template.tpl

```diff
@@ -1230,10 +1241,37 @@ const WRAPPER_NAMESPACE = '_amplitude';
 
 // Print a log message and set the tag to failed state
 const fail = msg => {
-  log(LOG_PREFIX + 'Error: ' + msg);
   return data.gtmOnFailure();
 };
 
+// Merge two Objects
+const mergeObject = (basedObject, overwriteObject) => {
```

Comment:
```suggestion
const mergeObject = (basicObject, overwriteObject) => {
```

basicObject or individualPropertiesObject based on the name you choose.

Reply:
Use basedObject/baseObject, because this is a general function to merge 2 objects which will overwrite the basedObject if there are any duplicate keys. it's not specific for individual properties. Let me know what you think.

Reply:
Yes `baseObject` is fine too.